src/Acr.UserDialogs/Acr.UserDialogs.csproj

```diff
@@ -16,7 +16,7 @@
         <LangVersion>latest</LangVersion>
         <NoWarn>$(NoWarn);1591;1701;1591;1702;1705;VSX1000;NU1603</NoWarn>
         <DisableExtraReferences>true</DisableExtraReferences>
-        <GenerateLibraryLayout Condition="$(TargetFramework.StartsWith('uap'))">true</GenerateLibraryLayout>
+        <GenerateLibraryLayout Condition=" $(TargetFramework.StartsWith('uap')) OR $(TargetFramework.StartsWith('net7')) ">true</GenerateLibraryLayout>
```

Comment:
.NET 7 is not released yet. I suppose you should replace it with net47

Reply:
Only net4 for forward compatibility

Reply:
ok, but not net7

Reply:
Fixed, thanks